src/app/components/Footer/styled.tsx

```diff
@@ -95,13 +103,21 @@ export const FooterWrapper = styled.footer`
   opacity: 1;
   transition: opacity 0.2s;
   ${disablePrint}
-  ${(props) => props.isVerticalNavOpen === false ? css`
-    @media (min-width: ${theme.breakpoints.desktopBreak}em) and ${contentWrapperAndNavWidthBreakpoint} {
-      padding-left: clamp(
-        0rem, calc(${toolbarWidth}rem - (100vw - ${contentWrapperMaxWidth}rem) / 2), ${toolbarWidth}rem
-      );
-    }
-  ` : ``}
+  ${props =>
+    props.isVerticalNavOpen === false
+      ? css`
+          @media (min-width: ${theme.breakpoints
+              .desktopBreak}em) and ${contentWrapperAndNavWidthBreakpoint} {
+            padding-left: clamp(
```

Comment:
The indentation here doesn't seem right, and to me, the original first line of the ternary reads much better without the extra line breaks and indentation.

Reply:
This is sort of a nightmare of nesting.
There are some things I disagree with prettier about, but at least it is consistent, and IMO it does more good than harm. We get a lot of dense code.